common/src/main/java/tachyon/client/FileSystemMasterClient.java

```diff
@@ -538,4 +513,71 @@ public synchronized int user_createDependency(List<String> parents, List<String>
   public synchronized DependencyInfo getDependencyInfo(int dependencyId) throws IOException {
     throw new UnsupportedOperationException("not implemented");
   }
+
+  /**
+   * Loads a file from the under file system.
+   *
+   * @param path the Tachyon path of the file
+   * @param recursive whether parent directories should be loaded if not present yet
+   * @return the file id
+   * @throws FileDoesNotExistException if the file does not exist
+   * @throws IOException if an I/O error occurs
+   */
+  public synchronized long loadFileInfoFromUfs(String path, boolean recursive)
+      throws IOException, FileDoesNotExistException {
+    int retry = 0;
+    while (!mClosed && (retry ++) <= RPC_MAX_NUM_RETRY) {
+      connect();
+      try {
+        return mClient.loadFileInfoFromUfs(path, recursive);
+      } catch (FileDoesNotExistException e) {
+        throw e;
+      } catch (TException e) {
+        LOG.error(e.getMessage(), e);
+        mConnected = false;
+      }
+    }
+    throw new IOException("Failed after " + retry + " retries.");
+  }
+
+  /**
+   * Mounts the given UFS path under the given Tachyon path.
+   *
+   * @param tachyonPath the Tachyon path
+   * @param ufsPath the UFS path
+   * @throws IOException an I/O error occurs
+   */
+  public synchronized boolean mount(TachyonURI tachyonPath, TachyonURI ufsPath) throws IOException {
+    int retry = 0;
+    while (!mClosed && (retry ++) <= RPC_MAX_NUM_RETRY) {
+      connect();
+      try {
+        return mClient.mount(tachyonPath.toString(), ufsPath.toString(), new MountOpts());
+      } catch (TException e) {
+        LOG.error(e.getMessage(), e);
+        mConnected = false;
+      }
+    }
+    throw new IOException("Failed after " + retry + " retries.");
+  }
+
+  /**
+   * Unmounts the given Tachyon path.
+   *
+   * @param tachyonPath the Tachyon path
+   * @throws IOException an I/O error occurs
+   */
+  public synchronized boolean unmount(TachyonURI tachyonPath) throws IOException {
+    int retry = 0;
+    while (!mClosed && (retry ++) <= RPC_MAX_NUM_RETRY) {
+      connect();
+      try {
+        return mClient.unmount(tachyonPath.toString());
```

Comment:
Let me remove the options here. I will introduce them in a follow up PR that uses options for all of the RPC calls.